WORKSPACE

```diff
@@ -16,6 +16,30 @@ workspace(name = "io_opentelemetry_cpp")
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
+# Load gRPC dependency
+# Note that this dependency needs to be loaded first due to
+# https://github.com/bazelbuild/bazel/issues/6664
+http_archive(
+    name = "com_github_grpc_grpc",
+    strip_prefix = "grpc-1.29.0",
+    urls = [
+        "https://github.com/grpc/grpc/archive/v1.29.0.tar.gz",
```

Comment:
Can we somehow make this work for CMake too?

Reply:
Do you mean build gRPC with CMake?

Reply:
I think it would be possible to build gRPC with CMake, but it would be quite involved. We could do something similar to https://github.com/open-telemetry/opentelemetry-cpp/pull/49

Reply:
I would love to see a CMake integration for this. The OTLP tests for Windows runs via CMake, so skipping CMake support would mean that those tests wouldn't be automatically run on Windows.

That might indeed be involved, and if that delays the remaining OTLP exporter work too much, let's at least write an issue so we can keep track of the remaining work (or somebody else can pick it up).

Reply:
Okay, thank you! I've added an issue here: https://github.com/open-telemetry/opentelemetry-cpp/issues/154